config/locales/en.yml

```diff
@@ -171,6 +171,128 @@ en:
       attached: is not attached
       content_type_invalid: image format not supported
       max_file_size_invalid: image over %{size}
+  expenses:
```

Comment:
I don't think this data should be in a locale file. If we translate to another language (see #240) then we would have to duplicate all of this to that locale file and keep them in sync.

Instead, this should live in `data/monies.yml` or similar and be parsed as YAML then spit out.

Also, let's calculate the totals in Ruby. I don't want to have to worry about updating three fields every time I add an expense or revenue line item.

Reply:
I think for this one, it will be better if we have access to the API. We can pull the info with a worker, do all the necessary calculations and then storing it in the DB.  Other than that, we can still use the YAML or a CSV if we just import and export of the current data. If so, which directory do you want the `monies.yml` or maybe `financials.yml`?

Reply:
My goal is to remove the external dependency entirely and make the entire thing static, outside of calculating the totals in Ruby.

Reply:
I moved the data for the table around. I am sure there is a better way to do this but for now this is what I came up with.